src/modules/auth/auth.route.ts

```diff
@@ -1,20 +1,22 @@
 import { FastifyInstance } from 'fastify';
 import AuthController from './auth.controller';
-import { $ref } from './auth.schema';
 import AuthService from './auth.service';
 import UserService from './user.service';
+import { JsonSchemaToTsProvider } from '@fastify/type-provider-json-schema-to-ts';
 
-export default async (fastify: FastifyInstance) => {
+export default async (Fastify: FastifyInstance) => {
 	const authController = new AuthController(new AuthService(), new UserService());
 
+	const fastify = Fastify.withTypeProvider<JsonSchemaToTsProvider>();
```

Comment:
Does this make any difference whatsoever?

Reply:
No... No it does not